tools/nixpkgs-hammer

```diff
@@ -172,7 +156,10 @@ def main(args):
                             link = false;
                         }} ]
                     else
-                        result.value.__nixpkgs-hammering-state.reports or [];
+                        [
+                          (result.value.__nixpkgs-hammering-state.reports or [])
```

Comment:
This no longer matches the type of the previous branches `List[NixpkgsHammeringMessage]`. Now it is `List[Union[List[NixpkgsHammeringMessage], str]]`. Maybe use `TypedDict('Result', {'report': List[NixpkgsHammeringMessage], 'position': Optional[str]})` for all branches?